R/transformCounts.R

```diff
@@ -308,6 +351,115 @@ setMethod("transformAssay", signature = c(x = "SummarizedExperiment"),
     return(mat)
 }
 
+####################################.calc_css###################################
+# This function applies cumulative sum scaling (CSS) to the abundance table.
+.calc_css <- function(mat, css.percentile = 0.5, scaling = 1000, ...) {
+    # Input check
```

Comment:
argument names could be just "percentile", "scaling" and "threshold"

Reply:
ACK.